Packs/Core/Playbooks/playbook-Ransomware_Enrich_and_Contain_README.md

```diff
@@ -0,0 +1,53 @@
+This playbook is responsible for alert data enrichment and response.
```

Comment:
This playbook is responsible for **ransomware** alert data enrichment and response.
(not any alert :) )